pkg/controllers/work/apply_controller.go

```diff
@@ -698,7 +812,8 @@ func (r *ApplyWorkReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		WithOptions(ctrloption.Options{
 			MaxConcurrentReconciles: r.concurrency,
 		}).
-		For(&fleetv1beta1.Work{}, builder.WithPredicates(predicate.GenerationChangedPredicate{})).
+		For(&fleetv1beta1.Work{}).
+		Watches(&fleetv1beta1.Work{}, eventHandler).
```

Comment:
This handler does not seem to filter out status change? That will be a lot more noises.

Reply:
added filter